tests/integration/states/pip.py

```diff
@@ -64,6 +64,16 @@ def test_pip_installed_errors(self):
                 shutil.rmtree(venv_dir)
 
```

Comment:
I think we have a with grains decorator I'm salt-testing, not that this approach is wrong...